src/base/preferences.cpp

```diff
@@ -1054,6 +1056,115 @@ void Preferences::setExecutionLogEnabled(bool b)
     setValue("Preferences/ExecutionLog/enabled", b);
 }
 
+bool Preferences::executionLogNormalType() const
```

Comment:
QVariant has generic fromValue(). QFlags is better since it allows to use something like Log::Normal | Log::Warning.